packages/database/lib/schema.ts

```diff
@@ -43,20 +56,29 @@ export const daos = pgTable('daos', {
   daoAcct: pubkey('dao_acct').primaryKey(),
   name: varchar('name').notNull(),
   url: varchar('url').notNull(),
+  xAccount: varchar('x_account'),
+  gitHub: varchar('github'),
+  description: text('description'),
   // In FaaS, each DAO is tied to its own token which futarchic markets will aim to pomp to the moon
-  mintAcct: pubkey('mint_acct').references(() => tokens.mintAcct).notNull()
+  mintAcct: pubkey('mint_acct').references(() => tokens.mintAcct).notNull(),
+  createdAt: timestamp('created_at').notNull().default(sql`now()`),
+  updatedAt: timestamp('updated_at').notNull().default(sql`now()`),
 });
 
 export const proposals = pgTable('proposals', {
   proposalAcct: pubkey('proposal_acct').primaryKey(),
   daoAcct: pubkey('dao_acct').references(() => daos.daoAcct).notNull(),
   proposalNum: bigint('proposal_num', {mode: 'bigint'}).notNull(),
+  title: varchar('title'),
+  description: varchar('description'),
+  categories: jsonb('categories'),
```

Comment:
What about third normal form!?

Reply:
Perhaps I want the ability to filter proposals by category for example. In that case a categories table is preferable

Reply:
> Perhaps I want the ability to filter proposals by category for example. In that case a categories table is preferable

I'd just create a view... Saves us some flexibility, but overall I don't mind creating another table...